src/Orleans/Messaging/BufferPool.cs

```diff
@@ -38,9 +39,11 @@ public string Name
             private set;
         }
 
-        internal static void InitGlobalBufferPool(IMessagingConfiguration config)
+        internal static void InitGlobalBufferPool(IOptions<MessagingOptions> messagingOptions)
         {
-            GlobalPool = new BufferPool(config.BufferPoolBufferSize, config.BufferPoolMaxSize, config.BufferPoolPreallocationSize, "Global");
+            var messagingOptionsValue = messagingOptions.Value;
```

Comment:
nit for future reference: I would just call this `options`, not `messagingOptionsValue`

Reply:
In extension methods I followed this, to make the body of those look like the same, but not locally. Once we set a standard here I agree to rename the ones where we're using only 1 type of options.